hardhat.config.js

```diff
@@ -11,16 +11,20 @@ for (const f of fs.readdirSync(path.join(__dirname, 'hardhat'))) {
 }
 
 const enableGasReport = !!process.env.ENABLE_GAS_REPORT;
+const enableProduction = process.env.COMPILE_MODE === 'production';
 
 /**
  * @type import('hardhat/config').HardhatUserConfig
  */
 module.exports = {
+  paths: {
+    sources: enableProduction ? 'contracts' : 'test/mocks',
+  },
```

Comment:
@cgewecke We tried to do this to remove `mocks` from our `contracts` directory, but as a consequence solidity-would only measure coverage in the mocks directory and ignoring the `contracts` directory.

I assume this is the same mechanism to avoid measuring coverage of dependencies in `node_modules`. Is there any way that we could make this setup work?